### PR TITLE
FIX Not download share link

### PR DESCRIPTION
**Pull request by leninrivas**

### CODE REVIEW

File: htdocs/core/lib/files.lib.php [hunk 1]
```diff
@@ -2913,14 +2913,15 @@ function dol_check_secure_access_document($modulepart, $original_file, $entity,
 		if ($fuser->rights->expedition->{$lire} || preg_match('/^specimen/i', $original_file)) {
 			$accessallowed = 1;
 		}
-		$original_file = $conf->expedition->dir_output."/".(strpos('sending/', $original_file) === 0 ? '' : 'sending/').$original_file;
-		//$original_file = $conf->expedition->dir_output."/".$original_file;
+		//$original_file = $conf->expedition->dir_output."/".(strpos('sending/', $original_file) === 0 ? '' : 'sending/').$original_file;
+		$original_file = $conf->expedition->dir_output."/".$original_file;
```
Comment (eldy) [inline htdocs/core/lib/files.lib.php:2917]: I don't understand the need to change this.
What is the value of $original_file you have that fails ?

Comment (leninrivas) [reply htdocs/core/lib/files.lib.php:2917]: Hi. It's duplicated sending.
sending/sending/ un Expedition and sending/receipt in delivery.

Comment (leninrivas) [reply htdocs/core/lib/files.lib.php:2917]: Its fix download from document.php?hashp= ... but crash download PDF from module. :(
This is the $fullpath_original_file: expedition/**sending/**receipt/T001-00000011/T001-00000011.pdf
That word is the problem.
In expedition is:
This is the $fullpath_original_file: expedition/**sending/**sending/EXP01-00000011/EXP01-00000011.pdf

**Comment by leninrivas** [comment]
Fixed in #24345. Closed

**Pull request closed without merging**